openid-4-verifiable-presentations-1_0.md

```diff
@@ -1537,34 +1537,11 @@ And lastly, as part of the request, the Wallet is provided with information abou
 
 ## Protocol
 
-The value of the `protocol` parameter of the W3C Digital Credentials API MUST be set to `openid4vp` for this profile.
+When used with the W3C Digital Credentials API [@!w3c.digital_credentials_api], the exchange protocol is the string `openid4vp` for this profile.
 
 ## Request {#browser_api_request}
 
-The `request` member of the W3C Digital Credentials API [@!w3c.digital_credentials_api] contains an OpenID4VP Authorization Request, where every OpenID4VP Authorization Request parameter is represented as a top-level JavaScript object member.
-
-The following is a non-normative example of how the W3C Digital Credentials API can be used with an unsigned OpenID4VP request when advanced security features of OpenID4VP are not used:
-
-```js
-try {
-  const credential = await navigator.identity.get({
-    digital: {
-      providers: [{
-        protocol: "openid4vp",
-        request:  {
-          response_type: "vp_token",
-          response_mode: "w3c_dc_api",
-          nonce: "n-0S6_WzA2Mj",
-          client_metadata: {...},
-          presentation_definition: {...}
-        }
-      }]
-    }
-  });
-} catch (err) {
-  // Handle errors and/or fallback to other invocation mechanisms
-}
-```
+The W3C Digital Credentials API [@!w3c.digital_credentials_api] contains an OpenID4VP Authorization Request, where every OpenID4VP Authorization Request parameter is represented as a top-level JavaScript object member.
```

Comment:
> where every OpenID4VP Authorization Request parameter is represented as a top-level JavaScript object member

Is this a new development in Digital Credentials API? How would this work for signed requests?

Reply:
The change here is removing "`request` member", I think that's okay, it's removing the coupling between the two specs but it doesn't change how signed requests are handled.

Reply:
is there any risk of this changing in the future? wouldn't this prevent passing multiple requests in an array? Would really prefer a text like `The Verifier MAY send all the OpenID4VP request parameters to the W3C Digital Credentials API as defined in [@!w3c.digital_credentials_api]`

Reply:
It's best to say less. We don't know what the final API will look like.